### PR TITLE
Update service graph with standard pod labels

### PR DESCRIPTION
**Pull request by mandarjog**
1. update service graph to use standard `app` label
2. fix prometheus virtual service with the new prometheus host

### CODE REVIEW

File: perf/load/templates/istio-ingress.gen.yaml [hunk 1]
```diff
@@ -26,10 +26,16 @@ spec:
   - route:
     - destination:
         host: {{ .Values.serviceNamePrefix }}0
-        subset: all
+        subset: v1
         port:
           number: 8080
-      weight: 100
+      weight: 30
+    - destination:
```
Comment (howardjohn) [inline perf/load/templates/istio-ingress.gen.yaml]: what's all this for? rest looks good

Comment (mandarjog) [reply perf/load/templates/istio-ingress.gen.yaml]: to have at least one traffic split in the default config.

**Pull request merged**